### PR TITLE
Fix parallelstore installation script on RHEL and Rocky Linux

### PR DESCRIPTION
**Pull request by wiktorn**
### Submission Checklist

Base image of RHEL 9 and Rocky Linux 9 GCP Optimized do not provide `/usr/bin/google_disable_automatic_updates` so execute it only if it exists.

The only image that provides that is HPC Linux.

Related: #2986

NOTE: Community submissions can take up to 2 weeks to be reviewed.

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cluster Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

**Comment by dgouju** [comment]
It impacts more images, including slurm-gcp-6-6-hpc-rocky-linux-8 for example. ParallelStore installation is almost globally broken.

File: modules/file-system/parallelstore/scripts/install-daos-client.sh [hunk 1]
```diff
@@ -52,7 +52,9 @@ EOF
 		fi
 
 		## TODO: Remove disable automatic update script after issue is fixed.
-		/usr/bin/google_disable_automatic_updates
+		if [ -x usr/bin/google_disable_automatic_updates ]; then
```
Comment (tpdownes) [inline modules/file-system/parallelstore/scripts/install-daos-client.sh:55]: @wiktorn by eye, there is a missing leading slash

**Review by tpdownes — DISMISSED** [review]

File: modules/file-system/parallelstore/scripts/install-daos-client.sh (hunk 1 above)
Comment (wiktorn) [reply modules/file-system/parallelstore/scripts/install-daos-client.sh]: Good catch, resolved!

**Comment by harshthakkar01** [comment]
> It impacts more images, including slurm-gcp-6-6-hpc-rocky-linux-8 for example. ParallelStore installation is almost globally broken.

Just to update on comment: We run daily tests on rocky and debian and its not broken. Had an offline discussion with [dgouju](https://github.com/dgouju). Let's not use parallelstore directly to nodeset, instead we should pass parallelstore to the controller and controller will mount it to compute nodes upon startup. 

see the referenced blueprint: https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/examples/ps-slurm.yaml#L82

**Review by harshthakkar01 — APPROVED** [review]

**Comment by harshthakkar01** [comment]
PR-test-ps-vm-debian (hpc-toolkit-dev) 
and
PR-test-ps-slurm (hpc-toolkit-dev) both passed on this PR.

**Pull request merged**